package/base-files/files/bin/config_generate

```diff
@@ -253,6 +253,12 @@ generate_static_system() {
 		add_list system.ntp.server='3.openwrt.pool.ntp.org'
 	EOF
 
+	local labelmac=$(get_mac_label)
+	if [ -n "$labelmac" ]; then
+		set system.@system[-1].hostname="OpenWrt-$(macaddr_geteui $labelmac)"
```

Comment:
Must be "uci set" here and below. I will fix it in my staging tree and test again.